src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -105,7 +117,20 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
     "enum" ~> typeName ~ ("{" ~> ambientEnumBody <~ "}") ^^ EnumDecl
 
   lazy val ambientEnumBody: Parser[List[Ident]] =
-    repsep(identifier <~ opt("=" ~ (numericLit | stringLit) ), ",") <~ opt(",")
+    repsep(identifier <~ opt("=" ~! valueExpression), ",") <~ opt(",")
+
+  lazy val valueExpression: Parser[_] =
+    rep1sep(success() ~>! rep(unaryValueOperator) ~
+      (numericLit | stringLit | identifierName | parenthesizedValueExpression), binaryValueOperator)
+
+  lazy val parenthesizedValueExpression =
+    "(" ~! valueExpression ~ ")"
+
+  lazy val binaryValueOperator =
+    elem("binary operator", tok => binaryValueOperators.contains(tok.chars))
```

Comment:
I'd rather you use a more direct style, like
```suggestion
    "+" | "-" | "*" | "/" | "**" | "%" | "<<" | ">>>" | ">>" | "&" | "|" | "^"
```

Reply:
may I ask why? in the current version the operators occur only once instead of twice at different locations